certi_builder/builder.py

```diff
@@ -79,6 +69,26 @@ def get_names(names_path: Text) -> List:
         logger.error(f"Error: {exception}")
 
 
+def underlineit(drawImage, name, location, text_color, font):
+    twidth, theight = drawImage.textsize(name, font=font)
```

Comment:
It seems you might have missed renaming your variable to a proper name. Please add this minor change

Reply:
changes applied.